lte/gateway/c/core/oai/tasks/s11/s11_tasks.cpp

```diff
@@ -356,8 +363,8 @@ static void* s11_mme_thread(void* args) {
 }
 
 //------------------------------------------------------------------------------
-status_code_e s11_mme_init(mme_config_t* mme_config_p) {
-  int ret = 0;
+extern "C" status_code_e s11_mme_init(mme_config_t* mme_config_p) {
+  status_code_e ret = RETURNok;
```

Comment:
ret is used only once instead we can directly return value

Reply:
Done